arena.js

```diff
@@ -0,0 +1,69 @@
+class Arena {
+  constructor (name, gladiators) {
+    this.name = name.charAt(0).toUpperCase() + name.slice(1).toLowerCase();
+    this.gladiators = []
+  }
+  addGladiator (gladiator){
+    this.gladiators.push(gladiator);
+    if (this.gladiators.length > 2){
+      this.gladiators.pop();
+      console.log(colosseum.gladiators);
+    }
+  }
+  addName (name){
+      this.name.push(name);
```

Comment:
since `this.name` is a String, you can't use `.push()` you must use an array. but the code above in `addGladiator` makes sense. Essentially, you are trying to do in `addName` what is already done in `addGladiator`.